cmd/fleetctl/mdm.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/service"
+	kithttp "github.com/go-kit/kit/transport/http"
+	"github.com/urfave/cli/v2"
+)
+
+func mdmCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "mdm",
+		Usage: "Run MDM commands against your hosts",
+		Flags: []cli.Flag{
+			configFlag(),
+			contextFlag(),
+			debugFlag(),
+		},
+		Subcommands: []*cli.Command{
+			mdmRunCommand(),
+		},
+	}
+}
+
+func mdmRunCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "run-command",
```

Comment:
Should there be an underscore alias for this?

Reply:
Ah, possibly yes, I'll make it the same as the other similar multi-words sub-commands.

Reply:
Added the `run_command` alias.